packages/zoe/src/contracts/loan/updateDebt.js

```diff
@@ -30,48 +31,72 @@ export const makeDebtCalculator = debtCalculatorConfig => {
     calcInterestFn = calculateInterest,
     originalDebt,
     loanMath,
-    periodAsyncIterable,
+    periodNotifier,
     interestRate,
+    interestPeriod,
     zcf,
     configMinusGetDebt,
   } = debtCalculatorConfig;
   let debt = originalDebt;
 
+  let lastCalculationUpdate;
```

Comment:
Since we only ever use the value, why not change this to:
```suggestion
  let lastCalculationTimestamp;
```

Reply:
done